slides/CM5/CM5_Tests_Variable_Selection.Rmd

```diff
@@ -314,8 +314,11 @@ $$
 }{
 \frac{(n-p) \hat{\sigma}^2}{\sigma^2} / (n-p)
 }
-&=
-\frac{1}{p\hat{\sigma}^2}\left(\hat{\boldsymbol{\beta}} - \boldsymbol{\beta}\right)^T (\mathbf{X}^T\mathbf{X})\left(\hat{\boldsymbol{\beta}} - \boldsymbol{\beta}\right)\\
+\qquad\qquad\qquad
```

Comment:
This equation already appears fully visible on my end, see compiled slide [here](https://hmma201.netlify.app/hmma201/slides/cm5/cm5_tests_variable_selection#17).
Does the linked slide appears ok to you ? If not, maybe that is a browser problem ? Otherwise, could it be a compilation issue ?
If the linked, already existing slide looks ok to you, then maybe this change is not necessary ?

Reply:
The equation on the linked slide doesn't appears fully for me. I tried other browsers (Brave, Internet Explorer and Firefox) and on another computer and I have the same problem. I had the same issue when I knit on Rstudio. I don't know where is the problem if it works for you.

Reply:
Ok, thank you for letting me know !
I don't really understand where the problem might come from, as the current version looks fine for me on several browsers and computers (all on mac though).
Anyway I'll keep that in mind for the future, and I'll try to stay safe and avoid long equations.
Your version looks fine on my computer too, so I'll merge your changes.
With your previous PR and this one, you've made the sum converge to 1 bonus point on your CC.
Thanks again !